### PR TITLE
Preserve necessary type and method

### LINKED ISSUE #50717: [wasm][tests][eat] System.Runtime.InteropServices.Tests
Seen [here](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-50657-merge-fbdd385f15244375b3/System.Runtime.InteropServices.Tests/console.8bd0e04f.log?sv=2019-07-07&se=2021-04-25T01%3A56%3A12Z&sr=c&sp=rl&sig=wYlbyPPmVwzBZv0VHWKKRXKSpRm%2FWQqGnMqpIrN0%2BEQ%3D)

```
[02:19:22] fail: [FAIL] System.Runtime.InteropServices.Tests.IDispatchImplAttributeTests.Ctor_ImplTypeShort(implType: -1)
[02:19:22] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:19:22] info:    at System.Runtime.InteropServices.Tests.IDispatchImplAttributeTests.Ctor_ImplTypeShort(Int16 implType)
[02:19:22] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:19:22] fail: [FAIL] System.Runtime.InteropServices.Tests.IDispatchImplAttributeTests.Ctor_ImplTypeShort(implType: 0)
[02:19:22] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:19:22] info:    at System.Runtime.InteropServices.Tests.IDispatchImplAttributeTests.Ctor_ImplTypeShort(Int16 implType)
[02:19:22] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:19:22] fail: [FAIL] System.Runtime.InteropServices.Tests.IDispatchImplAttributeTests.Ctor_ImplTypeShort(implType: 2)
[02:19:22] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:19:22] info:    at System.Runtime.InteropServices.Tests.IDispatchImplAttributeTests.Ctor_ImplTypeShort(Int16 implType)
[02:19:22] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:19:24] fail: [FAIL] System.Runtime.InteropServices.Tests.SetWin32ContextInIDispatchAttributeTests.Ctor_Default_ExistsInSrc
[02:19:24] info: Assert.NotNull() Failure
[02:19:24] info:    at System.Runtime.InteropServices.Tests.SetWin32ContextInIDispatchAttributeTests.Ctor_Default_ExistsInSrc()
[02:19:24] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:19:24] info: Finished:    System.Runtime.InteropServices.Tests.dll
```

### LINKED ISSUE #50721: [wasm][tests][eat] NRE in System.Text.Json.Serialization.Tests.CollectionTests
Seen in:
- [System.Text.Json.Tests](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-50657-merge-fbdd385f15244375b3/System.Text.Json.Tests/console.70a41acd.log?sv=2019-07-07&se=2021-04-25T01%3A56%3A12Z&sr=c&sp=rl&sig=wYlbyPPmVwzBZv0VHWKKRXKSpRm%2FWQqGnMqpIrN0%2BEQ%3D)

```
[02:25:39] fail: [FAIL] System.Text.Json.Serialization.Tests.CollectionTests.WriteHashSetTOfHashSetT
[02:25:39] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:25:39] info:    at System.Text.Json.Serialization.Tests.CollectionTests.WriteHashSetTOfHashSetT()
[02:25:39] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:25:40] fail: [FAIL] System.Text.Json.Serialization.Tests.CollectionTests.WriteArrayOfHashSetT
[02:25:40] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:25:40] info:    at System.Text.Json.Serialization.Tests.CollectionTests.WriteArrayOfHashSetT()
[02:25:40] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:25:40] fail: [FAIL] System.Text.Json.Serialization.Tests.CollectionTests.ReadISetTOfHashSetT
[02:25:40] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:25:40] info:    at System.Text.Json.Serialization.Tests.CollectionTests.ReadISetTOfHashSetT()
[02:25:40] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:25:40] fail: [FAIL] System.Text.Json.Serialization.Tests.CollectionTests.WriteISetTOfISetT
[02:25:40] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:25:40] info:    at System.Text.Json.Serialization.Tests.CollectionTests.WriteISetTOfISetT()
[02:25:40] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:25:40] fail: [FAIL] System.Text.Json.Serialization.Tests.CollectionTests.WriteISetTOfHashSetT
[02:25:40] info: System.NullReferenceException : Object reference not set to an instance of an object.
[02:25:42] info:    at System.Text.Json.Serialization.Tests.CollectionTests.WriteISetTOfHashSetT()
[02:25:42] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[02:30:19] info: Finished:    System.Text.Json.Tests.dll
```

### LINKED ISSUE #51911: [mono][eat] `System.Xml.XmlException : Resolving of external URIs was prohibited.`
Failing with `/p:EnableAggressiveTrimming=true`

[System.Xml.Xsl.XslCompiledTransformApi.Test](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-51697-merge-90ffb7de7255439c90/System.Xml.Xsl.XslCompiledTransformApi.Tests/console.50cf576c.log?sv=2019-07-07&se=2021-05-17T03%3A12%3A00Z&sr=c&sp=rl&sig=bXMAyTUXxvq97zAdN2V2kiBsJmBvk5%2BGJAHi0YFh1Ok%3D)

```
[03:37:36] fail: [FAIL] System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver3(param: "xmlResolver_document_function.txt", xslInputType: Navigator, readerType: XmlValidatingReader)
[03:37:36] info: System.Xml.Xsl.XslTransformException : An error occurred while loading document 'xmlResolver_document_function.xml'. See InnerException for a complete description of the error.
[03:37:36] info: ---- System.Xml.XmlException : Resolving of external URIs was prohibited.
[03:37:36] info:    at System.Xml.Xsl.Runtime.XmlQueryContext.GetDataSource(String uriRelative, String uriBase)
[03:37:36] info:    at System.Xml.Xsl.XmlILCommand.Execute(Object defaultDocument, XmlResolver dataSources, XsltArgumentList argumentList, XmlSequenceWriter results)
[03:37:36] info:    at System.Xml.Xsl.XmlILCommand.Execute(Object defaultDocument, XmlResolver dataSources, XsltArgumentList argumentList, XmlWriter writer)
[03:37:36] info:    at System.Xml.Xsl.XslCompiledTransform.Transform(XmlReader input, XsltArgumentList arguments, XmlWriter results, XmlResolver documentResolver)
[03:37:36] info:    at System.Xml.Xsl.XslCompiledTransform.Transform(String inputUri, String resultsFile)
[03:37:36] info:    at System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver3(Object param, XslInputType xslInputType, ReaderType readerType)
[03:37:36] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[03:37:36] info: ----- Inner Stack Trace -----
[03:37:36] info:    at System.Xml.XmlNullResolver.GetEntity(Uri absoluteUri, String role, Type ofObjectToReturn)
[03:37:36] info:    at System.Xml.Xsl.Runtime.XmlQueryContext.GetDataSource(String uriRelative, String uriBase)
```

List of failing tests:

```
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver2(param: "xmlResolver_document_function.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CTransformResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: URI, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Reader, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: Stream, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: Writer, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXmlResolverTest.XmlResolver3(param: "DefaultResolver.txt", xslInputType: Navigator, readerType: XmlValidatingReader, outputType: TextWriter, navType: XPathDocument)
fail: [FAIL] System.Xml.Tests.CXsltSettings.XsltSettings1_1_ExternalURI(param0: 5, param1: "XsltSettings.xml", param2: "XsltSettings2.xsl", param3: True, param4: False)
fail: [FAIL] System.Xml.Tests.CXsltSettings.XsltSettings1_1_ExternalURI(param0: 8, param1: "XsltSettings.xml", param2: "XsltSettings2.xsl", param3: True, param4: True)
fail: [FAIL] System.Xml.Tests.CXsltSettings.XsltSettings1_2_ExternalURI(param0: 18, param1: "XsltSettings.xml", param2: "XsltSettings2.xsl", param3: False, param4: False, param5: True, param6: False)
fail: [FAIL] System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver3(param: "xmlResolver_document_function.txt", xslInputType: Reader, readerType: XmlValidatingReader)
fail: [FAIL] System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver3(param: "xmlResolver_document_function.txt", xslInputType: URI, readerType: XmlValidatingReader)
fail: [FAIL] System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver3(param: "xmlResolver_document_function.txt", xslInputType: Navigator, readerType: XmlValidatingReader)
fail: [FAIL] System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver2(xslInputType: Reader, readerType: XmlValidatingReader)
fail: [FAIL] System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver2(xslInputType: URI, readerType: XmlValidatingReader)
fail: [FAIL] System.Xml.Tests.CTransformStrStrResolverTest.TransformStrStrResolver2(xslInputType: Navigator, readerType: XmlValidatingReader)

```

### LINKED ISSUE #53647: HTTP test null refs in 'Build Browser wasm Release AllSubsets_Mono_EAT' CI leg
```
[13:10:13] fail: [FAIL] System.Net.Http.Tests.HttpHeadersTest.NonValidated_ValidAndInvalidValues_DictionaryMembersWork
[13:10:13] info: System.NullReferenceException : Object reference not set to an instance of an object.
[13:10:13] info:    at System.Net.Http.Tests.HttpHeadersTest.NonValidated_ValidAndInvalidValues_DictionaryMembersWork()
[13:10:13] info:    at System.Reflection.RuntimeMethodInfo.Invoke(Object , BindingFlags , Binder , Object[] , CultureInfo )
[13:10:13] fail: console.error: /__w/1/s/src/mono/mono/utils/memfuncs.c:360 <disabled>
```
It doesn't repro in any of the other legs.  From @lewing: "Ah, so the EAT leg aggressively trims the test itself so this is almost certainly trimming related."

### PR DESCRIPTION
**Pull request by fanyang-mono**
Fixes #50721 
For test `System.Text.Json.Serialization.Tests.CollectionTests.WriteHashSetTOfHashSetT`,
the line of code which triggered System.NullReferenceException was https://github.com/xunit/assert.xunit/blob/3e9f13f2fae7ff8f379da0f19f72d56aea2dae46/Sdk/AssertEqualityComparer.cs#L249
Thus, `CompareTypedSets` was the missing method. It is a method of type `Xunit.Sdk.AssertEqualityComparer`, which belongs to assembly `xunit.assert.dll`

Fixes #50717
Rewrite the way of getting the type `System.Runtime.InteropServices.IDispatchImplAttribute`.

Fixes #53647 
Fix for #50721 also fixed this.

Fixes  #51911
Simply enable the tests. Not reproducible anymore.

### CODE REVIEW

**Comment by lewing** [comment]
cc @eerhardt

**Comment by radical** [comment]
Could you add the reasoning for these changes, like what depends on it, and how/what fails?

File: src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs [hunk 1]
```diff
@@ -12,7 +12,6 @@ public class IDispatchImplAttributeTests
         private const string ValueName = "Value";
 
         [Theory]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/50717", typeof(PlatformDetection), nameof(PlatformDetection.IsBuiltWithAggressiveTrimming), nameof(PlatformDetection.IsBrowser))]
```
Comment (MichalStrehovsky) [inline src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs:15]: If you rewrite the `Type type = typeof(HandleCollector).Assembly.GetType(TypeName);` line below (can't comment on that one directly) as `Type type = Type.GetType("System.Runtime.InteropServices.IDispatchImplAttribute, System.Runtime.InteropServices")` illink will be able to analyze this and the descriptor XML won't be needed.

Comment (fanyang-mono) [reply src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs:15]: Why it works this way, but not the original way?

File: eng/testing/ILLink.Descriptor.xunit.xml [hunk 2]
```diff
@@ -15,5 +15,10 @@
     <namespace fullname="Xunit" />
     <namespace fullname="Xunit.Sdk" />
   </assembly>
+  <assembly fullname="xunit.assert">
+    <type fullname="Xunit.Sdk.AssertEqualityComparer`1">
+      <method signature="System.Boolean CompareTypedSets(System.Collections.IEnumerable,System.Collections.IEnumerable)" />
```
Comment (eerhardt) [inline eng/testing/ILLink.Descriptor.xunit.xml:20]: Why is this necessary? Is xunit using Reflection to call this?

**Comment by eerhardt** [comment]
>         private const string TypeName = "System.Runtime.InteropServices.IDispatchImplAttribute";

Is `TypeName` still used?

---
Refers to: src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs:11 in e67171c. [](commit_id = e67171c21cf7d0ef0411e0b17d7b9ddd547f7f40, deletion_comment = False)

File: src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs (hunk 1 above)
Comment (eerhardt) [reply src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs:15]: It does seem like `typeof(XXX).Assembly.GetType(hard-coded string)` should be something the linker could recognize....

Comment (MichalStrehovsky) [reply src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/IDispatchImplAttributeTests.cs:15]: The illinker doesn't have the necessary intrinsics to understand the pattern. It wasn't deemed to be common enough to be in a situation where we both know the assembly and the name.

**Comment by MichalStrehovsky** [comment]
> Is TypeName still used?

Might be worth inlining `ValueName` as well. It feels like something that fell victim to "introduce a constant for the sake of introducing a constant". It's detrimental to readability.

File: eng/testing/ILLink.Descriptor.xunit.xml (hunk 2 above)
Comment (eerhardt) [reply eng/testing/ILLink.Descriptor.xunit.xml:20]: Sorry, I missed the original description, I see it is using reflection.

What do you think about also going and fixing xunit in this case? At a minimum we could add a `DynamicDependency` there.

Comment (MichalStrehovsky) [reply eng/testing/ILLink.Descriptor.xunit.xml:20]: https://github.com/xunit/assert.xunit/blob/58a59628853ed7a63895945fdac7984a92da2343/Sdk/AssertEqualityComparer.cs#L311-L313

From what I've seen in the past, attempting to annotate xUnit for trimming would be a health risk to the person doing it.

Comment (MichalStrehovsky) [reply eng/testing/ILLink.Descriptor.xunit.xml:20]: I would just add xunit.assert to the list above, together with Xunit and Xunit.Sdk to be sure.

Comment (fanyang-mono) [reply eng/testing/ILLink.Descriptor.xunit.xml:20]: Do you mean to preserve the whole `xunit.assert` assembly, instead of the specific method?

Comment (fanyang-mono) [reply eng/testing/ILLink.Descriptor.xunit.xml:20]: Thanks for the detailed explanation, @MichalStrehovsky. I was a little bit hesitate to preserve the whole `xunit.assert` assembly, because these tests would also be running with AOT on WASM. AOT things on WASM is quite slow currently. My current philosophy is to preserve as little as possible to make the test pass. That's being said, I am also curious if preserving the whole `xunit.assert` assembly could fix more trimming related test failures. I will open another PR to try it out. For this PR, I would like to keep it the minimal.

**Comment by fanyang-mono** [comment]
Failures of `Libraries Test Run release coreclr Linux_musl x64 Debug` are unrelated - #54778
Failures of `Build Android x64 Release AllSubsets_Mono_RuntimeTests` are unrelated - https://github.com/dotnet/xharness/issues/660
Failures of `Mono llvmaot Pri0 Runtime Tests Run Linux x64 release` are unrelated as well...

**Comment by fanyang-mono** [comment]
@eerhardt and @MichalStrehovsky Are the changes looks okay with you?

**Review by eerhardt — APPROVED** [review]

**Comment by fanyang-mono** [comment]
All tests has passed actually. Ready to be merged.

**Comment by stephentoub** [comment]
@fanyang-mono, this test is still disabled against one of the issues this closes:
https://github.com/dotnet/runtime/blob/a291fabdadee05b44aea09cc4bb11bef5a5dc305/src/libraries/System.Net.Http/tests/UnitTests/Headers/HttpHeadersTest.cs#L1541

**Comment by fanyang-mono** [comment]
> @fanyang-mono, this test is still disabled against one of the issues this closes:
> https://github.com/dotnet/runtime/blob/a291fabdadee05b44aea09cc4bb11bef5a5dc305/src/libraries/System.Net.Http/tests/UnitTests/Headers/HttpHeadersTest.cs#L1541

Oops, I didn't notice that there were more than one test methods associated with that issue. Will Open another PR to enable it. Thanks for your reminder.

**Comment by stephentoub** [comment]
Already did :-)

**Comment by fanyang-mono** [comment]
> Already did :-)

Thank you :)
